content/terraform/v1.13.x/docs/cli/import/usage.mdx

```diff
@@ -5,6 +5,12 @@ description: Learn now to use the `terraform import` command to import existing
 
 # Import existing resources
 
+<Note>
+
+The beta release of Terraform 1.14 lets you configure queries to search your existing infrastructure for resources and generate configuration to import them. Refer to the [Import existing resources in bulk](/terraform/language/v1.14.x/import/bulk) beta documentation for more information.
```

Comment:
```suggestion
Terraform `v1.14.x (beta)` implements a new workflow for searching existing infrastructure for resources, generating configuration, and importing them in bulk. Refer to the [Import existing resources in bulk](/terraform/language/v1.14.x/import/bulk) beta documentation for more information.
```
It might make sense to use a slightly different tact on this page.